tests/functional/adapter/test_simple_copy.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from dbt.tests.adapter.simple_copy.test_simple_copy import (
+    EmptyModelsArentRunBase,
+    SimpleCopyBase,
+)
+from dbt.tests.util import run_dbt
+
+
+@pytest.mark.iceberg
```

Comment:
Because one of the test's method is checking copy on materialized view, which is not supported on memory connector.
https://github.com/dbt-labs/dbt-core/blob/main/tests/adapter/dbt/tests/adapter/simple_copy/test_simple_copy.py#L81

on memory catalog:
```
io.trino.spi.TrinoException: This connector does not support creating materialized views
``